doc/content/modules/installation-guide/pages/how-tos/install.adoc

```diff
@@ -3,17 +3,41 @@
 This section delves into the detailed step-by-step instructions for installing {product}.
 Detailed explanations walk you through each step.
 
+== Full docker installation of {product}
+
 [INFO]
 ====
 Before you proceed with the installation, please ensure you have the following xref:requirements.adoc[prerequisites]:
 
-* xref:requirements.adoc[Java supported version]
-* https://www.docker.com/[Docker] installed, or an existing xref:requirements.adoc[supported database] installation with a user that has administrator privileges.
+* https://www.docker.com/[Docker] installed.
 ====
 
-== Get {product}
+{product} is distributed as a docker container.
+To install {product}, just download this https://github.com/eclipse-syson/syson/blob/{syson_tag}/docker-compose.yml[docker-compose] file and run the following docker command near the _docker-compose.yml_ file you've just downloaded:
+[source, bash]
+----
+docker compose up
+----
+
+It will install {product} (in the version corresponding to the tagged version of the documentation you are reading) and the postgres database in a composed container. 
```

Comment:
Delete "will" : This method installs {product} (matching the version tagged in this documentation) along with a PostgreSQL database, all within a Docker container.

Reply:
Done